tqdm/_tqdm.py

```diff
@@ -342,7 +353,7 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
                 bar_length, frac_bar_length = divmod(
                     int(frac * N_BARS * 10), 10)
 
-                bar = '#' * bar_length
+                bar = '█' * bar_length
```

Comment:
If you're making these changes for your own use only, don't make a "pull request" to merge it with master.  You can branch it and make changes and periodically merge the new changes to master into YOUR copy of the repo.